src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.DesignTime.targets

```diff
@@ -33,6 +33,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ProjectCapability Include="DynamicFileNesting" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
We don't get to change this file right now. VS is in escrow.

Reply:
Ok